test/performance/odp_l2fwd.c

```diff
@@ -2439,6 +2473,10 @@ static void print_options(void)
 		printf("interface default\n");
 	printf("Promisc mode:       %s\n", appl_args->promisc_mode ?
 					   "enabled" : "disabled");
+	if (appl_args->flow_control)
+		printf("Flow control:       %s%s\n",
+		       appl_args->pause_rx ? "RX " : "",
+		       appl_args->pause_tx ? "TX" : "");
```

Comment:
Nitpick, but maybe these could be lowercase.

Reply:
Changed in V2.